frontend/package.json

```diff
@@ -7,22 +7,27 @@
   "homepage": "/",
   "dependencies": {
     "@bcgov/bc-sans": "1.0.1",
+    "@pmmmwh/react-refresh-webpack-plugin": "^0.5.7",
     "@react-keycloak/web": "3.4.0",
     "@reduxjs/toolkit": "1.7.2",
+    "@svgr/plugin-svgo": "^6.2.0",
+    "@svgr/webpack": "^6.3.1",
     "@types/react-bootstrap-typeahead": "5.1.8",
     "@types/react-responsive": "8.0.5",
     "axios": "0.27.2",
     "bootstrap": "5.1.3",
     "classnames": "2.3.1",
     "crypto-js": "4.1.1",
+    "css-select": "^4.3.0",
     "dequal": "2.0.2",
     "formik": "2.2.9",
     "http-proxy-middleware": "2.0.3",
     "keycloak-js": "17.0.0",
     "leaflet": "1.7.1",
     "lodash": "4.17.21",
-    "moment": "2.29.1",
-    "moment-timezone": "0.5.34",
+    "moment": "^2.29.4",
```

Comment:
Normally you don't want to use the `^` version, as it will most likely be different versions used by every single build.

Reply:
Thanks, I think there is an npm config option I can set to not set a prefix when updating versions. I've committed a change to remove the prefix.

Reply:
save the exact version - `npm config set save-exact true`  
set a prefix - `npm config set save-prefix='~'`